template.rb

```diff
@@ -104,7 +104,17 @@ def add_template_repository_to_source_path
 
 say "=> Copying git configuration"
 directory "lib/install/.github", ".github"
-copy_file "lib/install/.gitattributes", ".gitattributes"
+gitattributes = <<-CODE
```

Comment:
Please use `~` instead of `-` and rename `CODE` to something meaninful

Reply:
Updated